src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -3,11 +3,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
-
-@EnableJpaRepositories("com.baeldung.persistence.repo")
```

Comment:
Just in case, these 2 annotations were making @Autowired not work. I assume the paramete to EnableJpa is the package it should include, right? Same for @EntityScan